src/themes/default/pages/Product.vue

```diff
@@ -350,17 +352,24 @@ export default {
     getInputName () {
       if (this.isSimpleOrConfigurable && !this.isProductLoading) { return this.$i18n.t('Quantity available', { qty: this.quantity }) }
       return this.$i18n.t('Quantity')
+    },
+    isAddToCartDisabled () {
+      return this.$v.$invalid || this.isProductLoading || (!this.quantity && this.isSimpleOrConfigurable)
     }
   },
   created () {
     this.getQuantity()
   },
-  mounted () {
-    this.$store.dispatch('recently-viewed/addItem', this.getCurrentProduct)
+  async mounted () {
+    await this.$store.dispatch('recently-viewed/addItem', this.getCurrentProduct)
+    await this.$store.dispatch('attribute/list', { // load filter attributes for this specific category
```

Comment:
Wasn’t it called in product/fetch? I bet it was :) wondering why there’s a need to call it in here? More over mounted() is not being called while clicking between recommended products (not switching the routes)